jbpm-services/jbpm-executor/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -10,6 +10,8 @@
     <mapping-file>META-INF/Executor-orm.xml</mapping-file>
     <class>org.jbpm.executor.entities.ErrorInfo</class>
     <class>org.jbpm.executor.entities.RequestInfo</class>
+    <class>org.jbpm.process.audit.ProcessInstanceLog</class>
```

Comment:
why is this here if the test is in test coverage ?

Reply:
The entities need to be mapped, otherwise they can't be found when we run the query.
I didn't get what you mean with:
> ... if the test is in test coverage